tests/runners/test_runner.py

```diff
@@ -60,3 +60,14 @@ def test_styles_runner_on_configured_repo(self):
         _, result = runner.workload()
 
         self.assertTrue(len(result) > 0)
+
+    def test_coverages_runner_on_configured_repo(self):
+        """Configured repo runner test."""
+        runner = PythonCoverageRunner(
+            "test", working_directory=self.configured_project_dir
+        )
+
+        runner.cloned_repo_directory = self.configured_project_dir
+        _, result = runner.workload()
+
+        self.assertTrue(len(result) > 0)
```

Comment:
Later on we will need to fix this tests by refactoring resource repositories. We need to have one repository which does not comply with tests, one which will comply, one which comply with config and one without.